src/createStore.ts

```diff
@@ -257,11 +257,7 @@ export default function createStore<S, A extends Action, Ext, StateExt>(
       throw new Error('Expected the nextReducer to be a function.')
     }
 
-    // TODO: do this more elegantly
```

Comment:
See line 84 for the "more elegant" approach, which allows any reducer to be assigned to `currentReducer`. It's all that can be done, really.